File: JavaScript/Exercicio24.js

function triangulo(numero) {
    conta = 1;
    for(i=1; i<= numero; i++) {
        for(j=1; j<= i; j++){
            process.stdout.write(`${conta}`)
            conta++
        }
        console.log(" ")
    }
}

triangulo(4)

File: CSharp/Exercicio24.cs

public void Triangulo(int numero)
{
    int conta = 1;
    for(int i= 1; i<= numero; i++)
    {
        for(int j = 1; j<=i; j++)
        {
            Console.Write($"{conta} ");
            conta++;
        }
        Console.WriteLine(" ");
    }
}

Triangulo(4);